src/python/pants/base/specs_parser_test.py

```diff
@@ -2,7 +2,10 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
```

Comment:
This file was rewritten. Same tests, but now use Pytest parameterization to reduce duplication.